app/src/main/java/com/keylesspalace/tusky/components/systemnotifications/NotificationFetcher.kt

```diff
@@ -109,8 +109,6 @@ class NotificationFetcher @Inject constructor(
 
                     // NOTE having multiple summary notifications this here should still collapse them in only one occurrence
                     notificationManagerCompat.notify(newNotifications)
-
-                    accountManager.saveAccount(account)
```

Comment:
Todo: Find what this line did

Reply:
I think it did nothing and remowing is save. `fetchNewNotifications` below does take care of saving the `notificationMarkerId`.